app/client/ui2018/modals.ts

```diff
@@ -623,10 +623,11 @@ export const cssModalBody = styled('div', `
 
 export const cssModalButtons = styled('div', `
   margin: 40px 0 0 0;
```

Comment:
Can you reuse this style in the document settings? Create a local:
`export const cssButtons = styled(cssModalButtons...`

This style is also used in various other places, and I'm not sure I caught all visual changes it might introduce.

Reply:
This modification seems uneeded after my last rebse